app/src/main/res/values/strings.xml

```diff
@@ -42,5 +45,5 @@
     <string name="minutes">minutes</string>
     <string name="seconds">seconds</string>
     <string name="resume_game">Resume Game</string>
-    <string name="restart_game">You have a previously saved game. Do you want to start a new game?</string>
+    <string name="restart_game">Do you want to override an already saved game?</string>
```

Comment:
`override`? Ain't this sound like a technical term to you? make this simpler

Reply:
Yes, I also think the same. But I have no solution for that as I want the message to be one-liner and very simple for users. I am open for suggestions.